Content.Shared/Mobs/Components/MobStateComponent.cs

```diff
@@ -13,30 +13,22 @@ namespace Content.Shared.Mobs.Components
     /// </summary>
     [RegisterComponent]
     [NetworkedComponent]
+    [AutoGenerateComponentState(true)]
     [Access(typeof(MobStateSystem), typeof(MobThresholdSystem))]
     public sealed partial class MobStateComponent : Component
     {
         //default mobstate is always the lowest state level
-        [ViewVariables] public MobState CurrentState { get; set; } = MobState.Alive;
+        [ViewVariables]
```

Comment:
if you changing these:
1. make this [DataField]
2. remove ViewVariables datafield is automatically vvable and readwrite
3. remove `"(allowedStates")` from next one its automatic